drinfomon.lic

```diff
@@ -699,6 +708,9 @@ while line = script.gets
         end
     elsif line =~ /^Circle: (\d+)/
       DRStats.circle = Regexp.last_match(1).to_i
+    elsif line =~ /^\[You're(.*\,)?(.*) (and|with).*\]$/
+      balance_map = ['completely imbalanced', 'hopelessly unbalanced', 'extremely imbalanced', 'very badly balanced', 'badly balanced', 'somewhat off balance', 'off balance', 'slightly off balance', 'solidly balanced', 'nimbly balanced', 'adeptly balanced', 'incredibly balanced']
+      DRStats.balance = balance_map.index(Regexp.last_match(2).strip) + 1
```

Comment:
Most of the appraisal scales go from 0->#, any reason to start balance at 1 instead of zero? Also I think there might be an alternate messaging form when you kill an enemy where you only get balance and no positional element, probably a pretty niche concern.